### DISCUSSION TITLE
Icon request: kolovrat

### DISCUSSION THREAD

**Discussion by vedranmiletic** [FortAwesome/Font-Awesome#21197] [Icon Requests]
### What can it be used for?

Kolovrat is one of the symbols used in [Slavic Native Faith (rodnovery)](https://en.wikipedia.org/wiki/Slavic_Native_Faith).

### Example image (optional)

Upper left part of [Rodnover symbols](https://commons.wikimedia.org/wiki/File:Rodnover_symbols.png) or [Red Rodnover kolovrat](https://commons.wikimedia.org/wiki/File:Red_Rodnover_kolovrat.svg).

### Icon request checklist

- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have included at least one example image if it could be helpful (optional)
- [X] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by kedmenecr** [comment]
I would love to have this too!

**Comment by davorrapic1** [comment]
this is great.  Gj vedranmiletic

**Comment by streger** [comment]
Please add this

**Comment by pinta83** [comment]
Please add

**Comment by tspinjac** [comment]
Please add.ty

**Comment by almirvuk** [comment]
I would love to have this too :)

**Comment by franraknic** [comment]
+1

**Comment by Atridij** [comment]
+1 Please add

**Comment by klenkovic** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
